src/app/assets/css/demo.css

```diff
@@ -0,0 +1,18 @@
+/* All styles created for demo purposes only, you may not need this for your project */
+.demo-button button {
+  margin-bottom: 5px; }
+
+.demo-icons li {
+  margin-bottom: 50px;
+  text-align: center; }
+
+.demo-icons .lnr,
+.demo-icons .fa {
+  font-size: 28px; }
```

Comment:
mark it in % for responsiveness else use a media query to distinguish  between screen and use them

Reply:
It was Already defined in the theme ... yes it is responsive